lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -29,7 +29,14 @@ define([
                             modified: '2016-03-25T13:45:21.220732Z',
                             marketing_slug: 'p_2?param=haha&test=b', 
                             id: 146,
-                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b'
+                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b',
+                            /* jshint ignore:start */
+                            banner_image_urls: [
+                                '348x116': 'http://www.edx.org/images/org1/test1',
+                                '435x145': 'http://www.edx.org/images/org1/test2',
+                                '726x242': 'http://www.edx.org/images/org1/test3'
+                            ]
+                            /* jshint ignore:end */
```

Comment:
We discussed and agreed earlier that naming the various image sizes by their dimensions will allow us to roll out additional sizes / versions of the same images in the future, in a backward-compatible manner.  I think we should stick with that.

I kind of think jshint is being overzealous in this case, in that these property names are not numeric, they just start with digits (although I get that there are times where you would not want that either).  Since jshint is only complaining about fixture data set up in test code, it really feels disproportionate to go back and change the server API just to placate it.  I vote for leaving things as they are unless you feel I am underestimating the risk @AlasdairSwan